### ISSUE TITLE
Given a certain level of zoom, some tracks freeze

### ISSUE THREAD

**Issue by beboche** [igvteam/igv.js#1617]
Hi,

thanks for developping such an amazing tool.

I'm embedding igv.js in a web app, and I just noticed that version 2.13.11 seems to suffer from a strange bug:

if you check this [page](https://mobidetails.iurc.montp.inserm.fr/MD/api/variant/48907/browser/#splicing) and scroll down to the igv browser, you will find out that:

- if you zoom out until 624 bp and try to scroll to the left, everything looks fine, in particular the 2 first bedgraphs tracks (the 3rd is empty)
- But if you zoom out to 1248bp, you will see that the 2nd track freezes,
- and to 2496, the 1st track also freezes. You can scroll right or left they do not follow. 
- The MANE and refseq tracks are fine.

I tested both Firefox 111.0.1 and brave browser 1.49.120 on  macOS ventura.

Another [example](https://mobidetails.iurc.montp.inserm.fr/MD/api/variant/5228/browser/#splicing).

I hope this is clear enough.

**Comment by beboche** [comment]
In addition to reported above, I have checked that version 2.13.10 does not suffer from the same bug and tht the firefox web tools report this error onload:

```
Uncaught (in promise) ReferenceError: lastNegValue is not defined
    draw https://path/to/igv.min.js??21311:70
    draw https://path/to/igv.min.js??21311??21311:38
    repaint https://path/to/igv.min.js??21311??21311:38
    updateViews https://path/to/igv.min.js??21311??21311:70
```

igv.min.js being here igv.2.13.11.min.js.

This message is absent from version 2.13.10

**Comment by jrobinso** [comment]
That bug was fixed in the current release, 2.15.0.    Please test against that version.

**Comment by beboche** [comment]
hi,
thanks, it's working correctly with 2.15.0.

Why is github showing the last release being 2.13.11? 
https://github.com/igvteam/igv.js/releases

**Comment by jrobinso** [comment]
Releases are through NPM. https://www.npmjs.com/package/igv.   The github "release" is an archive that sometimes lags.

BTW there was a parsing error affecting BED files that has been patched with release 2.15.1

**Comment by beboche** [comment]
ok thanks
